app/Http/Controllers/MemberController.php

```diff
@@ -26,15 +26,11 @@ public function getAdd(Request $request)
                 'messages'   =>$data['messages']
                 ], 200);
         }
-
-        DB::beginTransaction();
-        try {
-            Member::store($request);
-            DB::commit();
-            return $this->getList();       
-        } catch (Exception $e) {
-            DB::rollback();
-        }
+        Member::store($request);
```

Comment:
This is not how a model is supposed to be used, please look at: https://laravel.com/docs/5.4/eloquent#inserts for more information. Basically you can just get the values from the request object here and assign them to the model.